woocommerce/woocommerce-gateway-stripe
# Fix misaligned credit card icons

Fixes #1975

Internal thread p1632498482286700/1632453106.267900-slack-C02BHNRN6R1 

# Changes proposed in this Pull Request:

- Remove credit card icons from UPE payment method.
- Add an animation similar to UPE CC number input, displaying only one icon at a time for credit cards.

# Screenshots

## non-UPE

| Before | After |
| -- | -- |
| <img height="400" alt="WooCommerce Stripe non-UPE checkout form (before)" src="https://user-images.githubusercontent.com/5509901/134747809-faf622da-8331-4fa5-b52e-73e3b39c9c96.png"> | <img height="400" alt="WooCommerce Stripe non-UPE checkout form (after)" src="https://user-images.githubusercontent.com/5509901/134747503-7b2e9d0a-1e60-4d4a-ab28-d24089c348cb.gif"> |

## UPE

| Before | After |
| -- | -- |
| <img height="400" alt="WooCommerce Stripe UPE checkout form (before)" src="https://user-images.githubusercontent.com/5509901/134747801-1a397be0-5b47-4bd9-b4fb-9ed6fcce0cb3.png"> | <img height="400" alt="WooCommerce Stripe UPE checkout form (after)" src="https://user-images.githubusercontent.com/5509901/134747665-a49b7d08-a1f0-480e-b9f1-c8a8f0e2b145.png"> |

# Testing instructions

- Run `npm run sass`.
- Enable UPE.
- Add a product to the card and proceed to the checkout page.
- Ensure there's no credit card icon next to the payment method.
- Disable UPE.
- Ensure the credit card icon keeps changing between the supported credit card brands.
- Test on a couple different themes and ensure the icon is displayed consistently in comparison to other payment methods.
